icssc-projects/peterportal-client
# Course details partial

Completed redesign.

## Description
Created a shared component for course/professor popup.

## Steps to verify/test this change:
1. Go to search page
2. Click on a course or a professor
3. See the new design

## Todos:
- [ ] Aggregate reviews to show score.
